Tests/Data/Parabolic/HT/StaggeredCoupling/ADecovalexTHMCBasedHTExample/th_decovalex.prj

```diff
@@ -73,6 +70,21 @@
                             <type>Constant</type>
                             <value>0.0</value>
                         </property>
+                        <property>
+                            <name>density</name>
+                            <type>parameter</type>
+                            <parameter_name>rho_solid</parameter_name>
```

Comment:
For discussion:
Could be also
```xml
<property>
    <name>density</name>
    <parameter>rho_solid</parameter>
</property>
```

What do you think?
@TomFischer I know, the parsing is slightly more difficult.

Reply:
The advantage of your variant is a non-redundant, short description of a `ParameterProperty` object. The advantage of the implementation of the PR is that it is consistent with the specification of other parameter types.

@ @wenqing, @renchao-lu, @ErikNixdorf , @waltherm Opinions?

Reply:
I think that it is better to have the consistency with other types of input. Therefore I prefer to take the present one in this PR.

Reply:
In my mind, there are three types of property offered by far as follows: constant, linear dependency, and exponential dependency. I guess the initial design of the tag `<type>` intends to establish property dependency linking to primary process variables which will change over time. If a material property is not time-dependent, we assign the tag to constant. Otherwise, we assign to linear/exponential dependency. As to ParameterLib, I think the major business of ParameterLib is to tackle spatial heterogeneity, although it offers some time-associated functionalities. For this main reason, ParameterLib has its own tag for property type: function, group, meshnode, and meshelement (for spatial heterogeneity), curvescaled, and timedependentheterogeneousparameter (for time-dependency).

Reply:
I would vote for @TomFischer implementation, which seems more consistent with the existing, alternative options.

Reply:
Thanks for discussion.
:fast_forward: